mlops/docker_taxi/src/mlops_pipeline.py

```diff
@@ -77,252 +71,101 @@ def docker_taxi_data_regression(pipeline_job_input, model_name, build_reference)
-    except Exception as ex:
-        print(
-            "Oops! invalid credentials or error while creating ML environment.. Try again...",
-            ex,
-        )
-        raise
+        # demo how to change pipeline output settings
+        pipeline_job.outputs.pipeline_job_prepped_data.mode = "rw_mount"
+
+        return pipeline_job
 
 
 def prepare_and_execute(
-    build_environment: str,
-    wait_for_completion: str,
-    output_file: str,
+    model_name, build_environment, wait_for_completion, output_file
 ):
     """
-    Prepare and execute the MLOps pipeline.
+    Prepare and execute the pipeline.
 
     Args:
-        build_environment (str): environment name to execute.
-        wait_for_completion (str): "True" or "False" - indicates whether to wait for the job to complete.
-        output_file (str): The path to the output file where the job name will be written.
+        model_name (str): The name of the model.
+        build_environment (str): The build environment configuration.
+        wait_for_completion (str): Whether to wait for the pipeline job to complete.
+        output_file (str): A file to save the run ID.
     """
-    model_name = "docker_taxi"
-
     config = MLOpsConfig(environment=build_environment)
 
-    ml_client = MLClient(
-        DefaultAzureCredential(),
-        config.aml_config["subscription_id"],
-        config.aml_config["resource_group_name"],
-        config.aml_config["workspace_name"],
-    )
-
     pipeline_config = config.get_pipeline_config(model_name)
-
-    compute = get_compute(
-        config.aml_config["subscription_id"],
-        config.aml_config["resource_group_name"],
-        config.aml_config["workspace_name"],
-        pipeline_config["cluster_name"],
-        pipeline_config["cluster_size"],
-        pipeline_config["cluster_region"],
-    )
-
-    environment = get_environment(
-        config.aml_config["subscription_id"],
-        config.aml_config["resource_group_name"],
-        config.aml_config["workspace_name"],
-        env_base_image=config.environment_configuration["env_base_image"],
-        environment_name=pipeline_config["aml_env_name"],
-        docker_context_path=pipeline_config["docker_context_path"],
-        dockerfile_path=pipeline_config["dockerfile_path"],
-    )
-
-    print(f"Environment: {environment.name}, version: {environment.version}")
-
     published_model_name = generate_model_name(model_name)
-    published_experiment_name = generate_experiment_name(model_name)
-    published_run_name = generate_run_name(
-        config.environment_configuration["build_reference"]
-    )
 
-    pipeline_job = construct_pipeline(
-        compute.name,
-        f"azureml:{environment.name}:{environment.version}",
-        published_run_name,
-        build_environment,
-        config.environment_configuration["build_reference"],
-        published_model_name,
-        pipeline_config["dataset_name"],
-        ml_client,
+    pipeline = DockerTaxi(
```

Comment:
i propose renaming this to `pipeline_job_config`. since this is technically not the `pipeline_job` (that get's created from the job_config class method.

Reply:
i also propose this in the other `mlops/{model}/src/mlops_pipeline.py` files

Reply:
update variable names